.github/workflows/k8s-apply-sub.yaml

```
@@ -227,9 +231,16 @@ jobs:
       - name: Deploy
         env:
           CRONJOB_NAME: ${{ matrix.cronjob }}
-          K8S_NAMESPACE: ${{ inputs.environment }}-analytics
         run: |
           set -euo pipefail
 
           export PROJECT_DIR=$(pwd)
+
+          TEMPLATE=$(sh $SCRIPTS_FOLDER/helmTemplate-cron-single.sh --debug --environment ${{ inputs.environment }} --job $CRONJOB_NAME -i $PROJECT_DIR/commons/${{ inputs.environment }}/images.yaml -sd --output console)
+          echo $TEMPLATE
+
+          K8S_NAMESPACE=$(echo $TEMPLATE | yq 'select(.kind == "CronJob") | .metadata.namespace')
+          CRONJOB_FULLNAME=$(echo $TEMPLATE | yq 'select(.kind == "CronJob") | .metadata.name')
+  
+          echo "INFO - Apply changes for $CRONJOB_FULLNAME."
           $SCRIPTS_FOLDER/kubectlApply-cron-single-standalone.sh --debug --skip-dep --environment $K8S_NAMESPACE -j $CRONJOB_NAME -i $PROJECT_DIR/commons/${{ inputs.environment }}/images.yaml --output console 
```

Should we use the computed value here as well?

```suggestion
          $SCRIPTS_FOLDER/kubectlApply-cron-single-standalone.sh --debug --skip-dep --environment $K8S_NAMESPACE -j $CRONJOB_NAME -i $PROJECT_DIR/commons/$K8S_NAMESPACE/images.yaml --output console 
```